server/router/index.js

```diff
@@ -0,0 +1,16 @@
+const db = require('../database');
+
+const getRecordById = (req, res, next) => {
+  db.postgresql.getRecordById(req.params.id)
+    .then((data) => {
+      // console.log('getRecord data:', data);
+      res.status(200);
+      res.json(data.rows);
```

Comment:
Not a big deal, but you could maybe streamline this a little with `res.status(200).json(data.rows)`

Reply:
Thanks. Surely will do.